dev/com.ibm.ws.microprofile.faulttolerance.1.1_fat_tck/publish/tckRunner/tck/tck-suite-lite.xml

```diff
@@ -0,0 +1,85 @@
+<!-- Licensed under the Apache License, Version 2.0 (the "License"); you 
+    may not use this file except in compliance with the License. You may obtain 
+    a copy of the License at http://www.apache.org/licenses/LICENSE-2.0 Unless 
+    required by applicable law or agreed to in writing, software distributed 
+    under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES 
+    OR CONDITIONS OF ANY KIND, either express or implied. See the License for 
+    the specific language governing permissions and limitations under the License. -->
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="microprofile-faulttolerance-1.1-TCK" verbose="2"
+    configfailurepolicy="continue">
+
+    <test name="microprofile-faulttolerance 1.1 TCK">
+        <classes>
+        
+            <!-- Core functionality tests -->
+            <class name="org.eclipse.microprofile.fault.tolerance.tck.AsynchronousTest" />
+            <class name="org.eclipse.microprofile.fault.tolerance.tck.AsyncTimeoutTest">
+                <methods>
+                    <exclude name="testAsyncClassLevelTimeout"/>
+                </methods>
+            </class>
+            <class name="org.eclipse.microprofile.fault.tolerance.tck.ConfigTest">
+                <methods>
+                    <exclude name="testConfigMaxDuration"/>
+                    <exclude name="testClassLevelConfigMaxDuration"/>
+                </methods>
+            </class>
+            <class name="org.eclipse.microprofile.fault.tolerance.tck.CircuitBreakerTest">
+                <methods>
+                    <exclude name="testCircuitOpenWithMultiTimeouts"/>
+                    <exclude name="testCircuitOpenWithMultiTimeoutsAsync"/>
+                </methods>
+            </class>
+            <class name="org.eclipse.microprofile.fault.tolerance.tck.CircuitBreakerRetryTest" />
+            <class name="org.eclipse.microprofile.fault.tolerance.tck.FallbackTest">
+                <methods>
+                    <exclude name="testFallbackTimeout"/>
+                    <exclude name="testFallbacktNoTimeout"/>
+                </methods>
+            </class>
+            <class name="org.eclipse.microprofile.fault.tolerance.tck.RetryConditionTest" />
+            <class name="org.eclipse.microprofile.fault.tolerance.tck.RetryTest">
+                <methods>
+                    <exclude name="testRetryMaxDuration"/>
```

Comment:
Why do we exclude these?

Reply:
Are they too slow for lite?

Reply:
These methods from `RetryTest` actually have timing issues in the 1.1 TCK and are excluded even in FULL mode.

However, some other tests are excluded because they're slow and the functionality is adequately covered by other tests in the same class.